src/js/components/utils/useFetchIssues.tsx

```diff
@@ -29,11 +33,12 @@ export default ({
         dispatch,
       });
       setIssues(payload?.results || /* istanbul ignore next */ []);
+      setCurrentlyFetching(false);
     };
-    if (projectId && isOpen) {
+    if (projectId && isOpen && !currentlyResyncing) {
```

Comment:
@oluoluoxenfree Without this, it was successfully refreshing issues in the DB, but the front-end never got those newly-refreshed issues (until a page refresh). So this automatically triggers the refetch api calls once the resync is done.